main.go

```diff
@@ -68,6 +69,7 @@ func main() {
 	configureGlobalLogging(os.Getenv("GOLOGLEVEL"))
 	hlogChain := makeAccessLogChain(log.Logger)
 	rootMux := http.NewServeMux()
+	rootMux.Handle("/debug/vars", expvar.Handler())
```

Comment:
Technically, I don't think I need to do this since the docs say expvar can automatically register the route, but I chose to be explicit.